geoips/plugins/modules/output_checkers/geotiff.py

```diff
@@ -0,0 +1,104 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test script for representative product comparisons."""
+
+import subprocess
+import logging
+from os.path import splitext
+
+LOG = logging.getLogger(__name__)
+
+interface = "output_checkers"
+family = "standard"
+name = "geotiff"
+
+
+def correct_type(fname):
+    """Determine if fname is a geotiff file.
+
+    Parameters
+    ----------
+    fname : str
+        Name of file to check.
+
+    Returns
+    -------
+    bool
+        True if it is a geotiff file, False otherwise.
+    """
+    if splitext(fname)[-1] in [".tif"]:
+        return True
+    return False
+
+
+def outputs_match(plugin, output_product, compare_product):
+    """Use diff system command to compare currently produced image to correct image.
```

Comment:
One more stupid question, then I am good.
@jsolbrig @evrose54 are we good with the "required_methods" names of "correct_type" and "outputs_match" ?

Fine with me, just want to make sure we think about the names before we formalize/finalize them.

Reply:
The only change I would _maybe_ recommend is changing _correct_type_ to _correct_product_type_ or _correct_output_type_. Up to you though. I'm fine with either!

Reply:
Or "correct_file_type" since it's not like a python type or anything... I'll leave this one up to @jsolbrig , he is particular about names :-)

Reply:
@evrose54 per @jsolbrig , Update to "correct_file_format", and open an issue to investigate "filetype" module to use rather than exclusively checking file extensions for determining file types.

THEN YOU CAN RESOLVE AND WE CAN MERGE!